### PR TITLE
Added healthcheck API end-point

### PR DESCRIPTION
**Pull request by cobusc**
* Also pinned Django version 1.11.12. For some reason 1.11.3 breaks custom primary key "code" on country table.

### CODE REVIEW

File: authentication_service/views.py [hunk 1]
```diff
@@ -498,3 +503,21 @@ def get_success_url(self):
 
 class PasswordResetConfirmView(PasswordResetConfirmView):
     form_class = forms.SetPasswordForm
+
+
+class HealthCheckView(View):
```
Comment (AltusBarry) [inline authentication_service/views.py]: Set allowed methods on this view(`http_method_names=["get"]`), will stop anything other than GET being allowed.

**Comment by AltusBarry** [comment]
+1, just the one comment. Its more of a suggestion than anything else.

**Comment by AltusBarry** [comment]
+1, ignore comment. Base Django view already raises a method not allowed if the method was not added to the view.

File: authentication_service/views.py (hunk 1 above)
Comment (cobusc) [reply authentication_service/views.py]: As per your own comment below (copied here for reference): "Base Django view already raises a method not allowed if the method was not added to the view."

**Pull request merged**